client_sdk/python/sample.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+
+from bunkerhill_inference_api import InferenceAPIClient
+
+
+async def main():
+    async with InferenceAPIClient(
+        username="datashare-admin",
+        private_key_filename="private_key.pem",
+        base_url="https://api.coppshillhealth.com/",
```

Comment:
Consider changing the value of `base_url` to some other placeholder value. I don't think we want to expose the current value externally.